src/transit.js

```diff
@@ -1126,18 +1127,23 @@ class Transit {
 				} else {
 					chunks.push(chunk);
 				}
-				for (const ch of chunks) {
-					const copy = Object.assign({}, payload);
-					copy.seq = ++payload.seq;
-					copy.stream = true;
-					copy.data = ch;
 
-					this.logger.debug(`=> Send stream chunk to ${nodeID} node. Seq: ${copy.seq}`);
+				return this.Promise.all(
```

Comment:
@icebob It's not a good solution to not control the size of chunks, as this can lead to excessive CPU usage in the `Promise.all` call.

Reply:
what's the size which can cause problem?

Reply:
Based on my analysis, it seems most reasonable to derive the meaning from [`os.availableParallelism()`](https://nodejs.org/api/os.html#osavailableparallelism).

We are facing two issues: 
1. an unknown logic in `transitPublish` may cause synchronous blocking of operations in `Promise.all`.
2. We are unaware of the number of available system resources during the planning of `Promise.all` operations. We cannot control the peak load even if we had this information.

A small size of chunks for available resources is a bad solution, but for busy resources, it is a good solution.